test/ui-test/testSuites/suite_status/shared/steps/communitySteps.py

```diff
@@ -0,0 +1,21 @@
+
+from screens.StatusMainScreen import StatusMainScreen
+from screens.StatusCommunityPortalScreen import StatusCommunityPortalScreen
+from screens.StatusCommunityScreen import StatusCommunityScreen
+
+_statusCommunitityPortal = StatusCommunityPortalScreen()
+_statusMainScreen = StatusMainScreen()
+
+
+@Then("the user opens the community portal section")
+def step(context: any):
+    _statusMainScreen.goToCommunityPortal()
+
+@When("the user creates a community named |any|, with description |any|, intro |any| and outro |any|")
+def step(context, community_name, community_description, community_intro, community_outro):
+    _statusCommunitityPortal.createCommunity(community_name, community_description, community_intro, community_outro)
+
+
+@Then("the user lands on the community")
+def step(context): 
+    StatusCommunityScreen()
```

Comment:
Maybe we can extend the verification here checking that the introduced params, like the `name` and `description` are the ones that we expect. 

**NOTE:** To pass data accross steps you can use `context` object.